buildlib/jucx-publish.yml

```diff
@@ -0,0 +1,49 @@
+parameters:
+  target: publish
+  temp_cfg: $(System.DefaultWorkingDirectory)/bindings/java/src/main/native/build-java/tmp-settings.xml
+  gpg_dir: $(System.DefaultWorkingDirectory)/bindings/java/src/main/native/build-java/gpg
+
+steps:
+  - bash: |
+      set -eE
+      {
+        echo -e "<settings><servers><server>"
+        echo -e "<id>ossrh</id><username>\${env.SONATYPE_USERNAME}</username>"
+        echo -e "<password>\${env.SONATYPE_PASSWORD}</password>"
+        echo -e "</server></servers></settings>"
+      } > ${{ parameters.temp_cfg }}
+    displayName: Generate temporary config
+
+  - task: DownloadSecureFile@1
+    displayName: Download Secure file
+    inputs:
+      secureFile: sparkucx-secret.gpg
+    name: privateKey
+
+  - task: DownloadSecureFile@1
+    displayName: Download Secure file
+    inputs:
+      secureFile: sparkucx-public.gpg
+    name: publicKey
+
+  - bash: |
+      set -eE
+      TAG=`git describe --tags`
+      make -C bindings/java/src/main/native/ set-version JUCX_VERSION=${TAG:1}
```

Comment:
the default version is "1" ?

Reply:
No, TAG=`v1.8.0-rc2` `${TAG:1}=1.8.0-rc2`.  ${TAG:1} - removes 1 character in string

Reply:
why not v1.8.0-rc2?

Reply:
Maven versioning: https://docs.oracle.com/middleware/1212/core/MAVEN/maven_version.htm#MAVEN8855
Requires version to be in form: MAJOR_VERSION.MINOR_VERSION,etc.

Reply:
ok, pls add comment in yml